README.md

```diff
@@ -1306,7 +1306,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Sheharyar Khan](https://github.com/Sheharyark)
   - [Sheheer Ali](https://github.com/sheheeralizx)
   - [Sherelle Scott](https://github.com/Sherelle-S)
-  - Shelby John
+  - [Shelby John](https://github.com/johnshelb)
```

Comment:
Your name must be in alphabetical order [Line 1308] as "L" letter comes before "R"